packages/aws_common/lib/src/io/aws_file_platform_io.dart

```diff
@@ -104,4 +106,14 @@ class AWSFilePlatform extends AWSFile {
     // the constructors ensures it won't reach to this point, but just in case
     throw const InvalidFileException();
   }
+
+  @override
+  String? get contentType {
```

Comment:
Can we make this `late final` so it's cached the first time if called multiple times?

Reply:
Same Q for web